packages/amplify-migration-tests/src/__tests__/update_tests/storage_migration_update.test.ts

```diff
@@ -1,132 +0,0 @@
-import { $TSObject } from 'amplify-cli-core';
-import {
-  addAuthWithDefault,
```

Comment:
I see some of these tests are about other categories not under data's umbrella, are these migration tests about transformer migration or project migration? Just want to make sure we're not taking tests that we don't own. (whole amplify-migration-tests package)

Reply:
You're right. We need only API parts of this. Updating the split script. Thanks